src/main/java/com/epam/test_generator/services/SuitService.java

```diff
@@ -1,17 +1,25 @@
 package com.epam.test_generator.services;
 
-import static com.epam.test_generator.services.utils.UtilsService.checkNotNull;
-
 import com.epam.test_generator.dao.interfaces.CaseVersionDAO;
 import com.epam.test_generator.dao.interfaces.SuitDAO;
 import com.epam.test_generator.dto.SuitDTO;
+import com.epam.test_generator.dto.SuitRowNumberUpdate;
 import com.epam.test_generator.entities.Suit;
+import com.epam.test_generator.services.exceptions.BadRequestException;
```

Comment:
BadRequest is unused import. I would rather use BadRequestException instead of IllegalArgumentException. Former is already defined in GeneralExceptionHandler, while IllegalArgumentException is very general exception and may be throw by most of libraries.